package exceptions;

public class ContactAlreadyExistsException extends Throwable{

    public void sendError() {
        System.out.println("error");
    }
}
